src/v/cluster/self_test_rpc_types.h

```diff
@@ -179,6 +179,59 @@ struct netcheck_opts
     }
 };
 
+struct cloudcheck_opts
+  : serde::
+      envelope<cloudcheck_opts, serde::version<0>, serde::compat_version<0>> {
+    // Descriptive name given to test run
+    ss::sstring name{"Cloud credentials check"};
+
+    // Timeout duration for cloud storage requests.
+    ss::lowres_clock::duration timeout{std::chrono::milliseconds(5000)};
+
+    // Backoff duration for cloud storage requests.
+    ss::lowres_clock::duration backoff{std::chrono::milliseconds(10)};
+
+    // Amount of fibers to run per shard
+    uint16_t parallelism{10};
```

Comment:
this seems to be unused, did i miss it? separately, is it is desired for the cloud self-test to perform a benchmark vs only verify that connectivity and basic operations are working? i'm not sure what the right answer is.

Reply:
Yes, currently this is unused- I also had the open question of whether more rigorous benchmarking across multiple shards was desired, or whether just verifying connectivity with cloud storage was ideal.